website/docs/r/nsxt_network_dhcp.html.markdown

```diff
@@ -56,34 +43,12 @@ resource "vcd_nsxt_network_dhcp" "pools" {
 }
 ```
 
-## Example Usage 2 (Pool removal on VCD 10.1)
-
-DHCP pool definitions cannot be removed once defined therefore the trick is to recreate parent Org VDC network.
-This can be done by following such procedure:
-
-* Define the network and DHCP pools as in [Example Usage 1](#example-usage-1)
-* Use `terraform taint` on the parent network to force recreation:
-   ```sh
-   # terraform taint vcd_network_routed_v2.parent-network
-   Resource instance vcd_network_routed_v2.parent-network has been marked as tainted.
-   ```
-* Modify/remove `vcd_nsxt_network_dhcp` definition as per your needs
-* Perform `terraform apply`. This will recreate tainted parent Org VDC network and new DHCP pools if defined.
-You will see a WARNING during removal but it will not break :
-```sh
-vcd_nsxt_network_dhcp.pools: Destroying... [id=urn:vcloud:network:209666ec-6253-418a-a816-076b15413fea]
-vcd_nsxt_network_dhcp WARNING: for VCD versions < 10.2 DHCP pool removal is not supported. Destroy is a NO-OP operation for VCD versions < 10.2. Please recreate parent network to remove DHCP pools.
-vcd_nsxt_network_dhcp.pools: Destruction complete after 0s
-```
-
```

Comment:
Why are the two sections above removed all-together?

Reply:
Mainly I did the removal because we don't support 10.1 anymore, but probably the sentence about 10.2 stil deserves a note. Checking once more now.

Reply:
Thank you. It's actually very handy that Terrraform docs are there for all provider versions, so users still using older versions still see the actual details.